0003-mitigate-dns-rebinding-attacks-against-daemon.patch

```diff
@@ -1,286 +0,0 @@
-From 440f03e9ad5ed3efc9e7d943befe33e5cbb65a20 Mon Sep 17 00:00:00 2001
```

Comment:
You'll also need to remove the reference to this file from the JSON. Have you tried building this branch?